src/systems/user_commands/UserCommands.cc

```diff
@@ -1823,13 +1823,23 @@ bool VisualCommand::Execute()
     return false;
   }
 
-  if (visualMsg->id() == kNullEntity)
+  Entity visualEntity = kNullEntity;
+  std::string entityName = visualMsg->name();
+  if (!entityName.empty())
+  {
+    visualEntity =
+        this->iface->ecm->EntityByComponents(components::Name(entityName));
+  }
+  else if (visualMsg->id() != kNullEntity)
```

Comment:
It's better to check the ID before the name. IDs are unique, while names aren't. The name usually needs to be used in conjunction with the `parent_name` or `parent_id` to narrow down what the exact entity.

Reply:
I used `parent_name`  bb7f37e0289cdde223571b2444a7be71594a55f0